cmd/timoni/build.go

```diff
@@ -141,7 +142,9 @@ func runBuildCmd(cmd *cobra.Command, args []string) error {
 
 	buildResult, err := builder.Build()
 	if err != nil {
-		return fmt.Errorf("build failed, error: %w", err)
+		return fmt.Errorf("build failed:\n%s", errors.Details(err, &errors.Config{
```

Comment:
I think we need this in apply, also bundle apply.

Reply:
Yes, I was tempted to change it everywhere but wanted to keep the first PR simple.

Reply:
I've update the PR, could you please take another look? :)